staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -159,7 +159,7 @@ func (d *Helper) EvictPod(pod corev1.Pod, evictionGroupVersion schema.GroupVersi
 			},
 			DeleteOptions: &delOpts,
 		}
-		return d.Client.PolicyV1().Evictions(eviction.Namespace).Evict(context.TODO(), eviction)
+		return d.Client.PolicyV1().Evictions(eviction.Namespace).Evict(d.getContext(), eviction)
```

Comment:
I don't think, this could lead to any bugs because `Ctx` is always nil.

Reply:
It seems that not using in `k/kubectl` indeed, but maybe it can be import as a lib?

Additional, I think it's probably fine because the method guarente this can not be nil , and `d.getContext()` was used in early release.  

https://github.com/kubernetes/kubernetes/blob/4aecb151b8301a6fc69f63c08f640c7b2b4289fc/staging/src/k8s.io/kubectl/pkg/drain/drain.go#L458-L464